cognite_toolkit/cdf_tk/load.py

```diff
@@ -728,6 +752,30 @@ def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> FileMetadata
             )
         except Exception:
             files = FileMetadataList.load(load_yaml_inject_variables(filepath, ToolGlobals.environment_variables()))
+        if len(files.data) == 1:
+            # If we have a file with exact one file config, check to see if this is a pattern to expand
+            if "$FILENAME" in files.data[0].external_id or "":
+                # It is, so replace this file with all files in this folder using the same data
+                file_data = files.data[0]
+                ext_id_pattern = file_data.external_id
+                files = FileMetadataList([])
+                for file in Path(filepath.parent).glob("*"):
```

Comment:
Parent of a path is already a path
```suggestion
                for file in filepath.parent.glob("*"):
```